<launch>
    <node name="calibrate_angular" pkg="miiboo_bringup" type="check_angular.py" output="screen">
        <param name="test_angle"     value="360.0"/><!-- unit:degree -->
        <param name="speed"          value="0.5"/><!-- unit:rad/s -->

	    <param name="base_frame"     value="/base_footprint"/>
        <param name="odom_frame"     value="/odom"/>
    </node>
</launch>
